wagtail/images/templatetags/wagtailimages_tags.py

```diff
@@ -107,3 +111,14 @@ def render(self, context):
             for key in self.attrs:
                 resolved_attrs[key] = self.attrs[key].resolve(context)
             return rendition.img_tag(resolved_attrs)
+
+
+@register.simple_tag()
+def image_url(image, filter_spec, viewname='wagtailimages_serve'):
+    try:
+        return generate_image_url(image, filter_spec, viewname)
+    except NoReverseMatch:
+        raise ImproperlyConfigured(
+            "'image_url' tag requires the " + viewname + " view to be configured. Please see "
+            "http://docs.wagtail.io/en/latest/advanced_topics/images/image_serve_view.html#setup for instructions."
```

Comment:
I'd suggest linking to the stable version of the docs here:
```suggestion
            "https://docs.wagtail.io/en/stable/advanced_topics/images/image_serve_view.html#setup for instructions."
```

Reply:
Done